test/controllers/admin/settings_controller_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class Admin::SettingsControllerTest < ActionController::TestCase
+
+  setup do
+    session[:user_id] = User.find_by(username: "adam").id
```

Comment:
True I just feel more confident when the test pass by querying the actual database